.github/workflows/build-lint-test.yml

```diff
@@ -24,19 +24,22 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v3
 
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 'latest'
+
       - name: Use Node.js
         uses: actions/setup-node@v3
         with:
           node-version: '18'
 
-      - name: Install deps and build (with cache)
-        uses: bahmutov/npm-install@v1
+      - run: pnpm install
 
       - name: Lint
-        run: yarn lint
+        run: pnpm lint
 
       - name: Build
-        run: yarn build
+        run: pnpm build
 
       - name: Test
-        run: yarn test --ci --maxWorkers=2
+        run: pnpm run test --ci --maxWorkers=2
```

Comment:
I don't think we need `run` in the commands

Reply:
I also don't think so, but locally when running `pnpm test --ci --maxWorkers=2` I get this error message `ERROR   ERROR  Unknown options: 'ci', 'maxWorkers'`, I wanted to test it with GitHub Actions, but the test fails before.